apps/docs/scripts/lib/generateApiContent.ts

```diff
@@ -7,7 +7,6 @@ const { log: nicelog } = console
 export async function generateApiContent(): Promise<GeneratedContent> {
 	const articles: Articles = {}
 	const CONTENT_DIRECTORY = path.join(process.cwd(), 'content')
-	// eslint-disable-next-line @typescript-eslint/no-var-requires
```

Comment:
I saw that some of these disables are dropped, but still feels like they could  be useful in other places?